sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -40,6 +40,7 @@
   <!-- Set to true to spawn the position controllers for the arm-->
   <arg name="arm_position" default="$(eval not arm_trajectory)"/>
   <arg name="arm_speed_scale" default="0.5"/>
+  <arg name="kinematics_config" default="$(find sr_ur_calibration)/calibrations/bristol_ur10_calibration.yaml"/>
```

Comment:
I think we don't want to introduce dependencies on sr_ur_calibration.
Shouldn't this be pointing to the default calibration?

Reply:
Yes, good spot https://github.com/shadow-robot/sr_interface/pull/446/commits/dfc48c2ad7b41e83a4c615b3f234232e4b8d25a3